test/internet/test-dns-ipv6.js

```diff
@@ -40,6 +40,11 @@ function checkWrap(req) {
   assert.ok(typeof req === 'object');
 }
 
+if (!common.hasIPv6) {
+  console.log('1..0 # Skipped: ipv6 part of test, no IPv6 support');
```

Comment:
the entire test is ipv6. You might want to make copy specify the entire test has failed